docs/source-pytorch/conf.py

```diff
@@ -28,6 +28,8 @@
 # VARIABLES WHEN WORKING ON DOCS... MAKE THIS TRUE TO BUILD FASTER
 # -----------------------
 _PL_FAST_DOCS_DEV = bool(int(os.getenv("PL_FAST_DOCS_DEV", 0)))
+_DOCS_FETCHING_ASSETS = int(os.environ.get("DOCS_FETCHING_ASSETS", True))
```

Comment:
Why not include this under FAST_DOCS_DEV? Do we need one more flag?

Reply:
I would keep it separate as it does different things...

Reply:
ok, reverted...